draft-ietf-rtcweb-jsep.xml

```diff
@@ -405,6 +405,23 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
+          
+	  <t> RTP CNAME values provide a canonical name for the RTP
+	  endpoint, allowing other RTP endpoints to determine which
+          RTP streams are using the same clock and thus which clock
+	  sources can be used that to synchronize media playout. </t>
+
+	  <t> Any MediaStreamTracks which have different clock sources
```

Comment:
I think it would be a good idea to include some of the 'why' behind these statements.